src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -89,30 +85,72 @@ public static void AddTodoTemplateSwaggerGen(this IServiceCollection services)
     {
         services.AddSwaggerGen(options =>
         {
-            options.AddSecurityDefinition("bearerAuth", new OpenApiSecurityScheme
-            {
-                Name = "Authorization",
-                Type = SecuritySchemeType.Http,
-                Scheme = "bearer",
-                BearerFormat = "JWT",
-                In = ParameterLocation.Header,
-                Description = "JWT Authorization header using the Bearer scheme."
-            });
+            options.AddSecurityDefinition("bearerAuth",
+                new OpenApiSecurityScheme
+                {
+                    Name = "Authorization",
+                    Type = SecuritySchemeType.Http,
+                    Scheme = "bearer",
+                    BearerFormat = "JWT",
+                    In = ParameterLocation.Header,
+                    Description = "JWT Authorization header using the Bearer scheme."
+                });
 
             options.AddSecurityRequirement(new OpenApiSecurityRequirement
             {
                 {
                     new OpenApiSecurityScheme
                     {
-                        Reference = new OpenApiReference
-                        {
-                            Type = ReferenceType.SecurityScheme,
-                            Id = "bearerAuth"
-                        }
+                        Reference = new OpenApiReference { Type = ReferenceType.SecurityScheme, Id = "bearerAuth" }
                     },
-                    new string[] {}
+                    new string[] { }
                 }
             });
         });
     }
+
+    public static void AddHealthChecks(
+        this IServiceCollection services,
+        IConfiguration configuration,
+        IWebHostEnvironment environment)
+    {
+        var appsettings = configuration.GetSection(nameof(AppSettings)).Get<AppSettings>();
+
+        var healthCheckSetting = appsettings.HealCheckSettings;
+        
+        if (!healthCheckSetting.EnableHealthChecks)
+            return;
+
+        var emailSettings = appsettings.EmailSettings;
+
+        services
+            .AddHealthChecksUI(setupSettings: setup =>
+            {
+                setup.AddHealthCheckEndpoint("BitHealthCheck",
+                    environment.IsDevelopment() ? "http://localhost:5000/healthz" : "/healthz");
```

Comment:
is this check required? can't we just simply use `/healthz` for all of our environments?
@ysmoradi

Reply:
I tested that and got an exception, but now tried again and it's working. I assume this because of misconfiguration at that moment. I fix it in next commit

Reply:
That's great, so please remove all of the `environment` argument passing between methods that you've implemented too.